style.css

```diff
@@ -0,0 +1,21 @@
+body{
+    margin: 2rem;
+    text-align: center;
+}
+
+header{
+text-align: center;
+background-color: #BF2886;
+color: white;
+}
+.imglogo{
+    width: 150px;
```

Comment:
using pixels to size an image is not recommended. Usually you want to make the image responsive. If you must use a specific image size, use `rem` or `em` units to make sizing easier on mobile/tablet versions